packages/api/tests/endpoints/test-distribution.js

```diff
@@ -46,6 +54,18 @@ const bucketMap = {
         'Content-Type': 'text/plain',
       },
     },
+    path2: {
+      bucket: 'bucket-path-2',
+      headers: {
```

Comment:
Since we support any headers, I suggest to remove these specific header names, just leave a couple like Content-Type and custom-header

Reply:
Great suggestion! Done.